packages/@aws-cdk/aws-lambda-destinations/lib/lambda.ts

```diff
@@ -1,22 +1,65 @@
+import * as events from '@aws-cdk/aws-events';
+import * as targets from '@aws-cdk/aws-events-targets';
 import * as lambda from '@aws-cdk/aws-lambda';
 import { Construct } from '@aws-cdk/core';
+import { EventBridgeDestination } from './event-bridge';
+
+/**
+ * Options for a Lambda destination
+ */
+export interface LambdaDestinationOptions {
+  /**
+   * Whether the destination function receives only the `responsePayload` of
+   * the origin function.
+   *
+   * @default false The destination function receives the full invocation record.
+   */
+  readonly responseOnly?: boolean;
+}
```

Comment:
Add an entry in the README for this explaining this further. 

I can't seem to find the documentation that explains what the schema of the message would be, besides [this one](https://docs.aws.amazon.com/lambda/latest/dg/invocation-async.html#invocation-async-destinations).

We should explain with an example what the events look like when this options is set vs unset. Also, how this would differ when the destination is set for `onSuccess` vs. `onFailure`.

Reply:
There's nothing more than the doc you are referring to and the [blog post](https://aws.amazon.com/blogs/compute/introducing-aws-lambda-destinations/)

Reply:
More of a reason to consolidate into our README.